tests/lib/testing_wrapper.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import six
+import json
+import pytest
+import tempfile
+import shutil
+
+from tests.lib.db_handler import DBHandler
+from tests.lib.file_handler import RemoteFileHandler
+
+
+class TestCase:
```

Comment:
I would create something like `BaseTestCase` without any attributes. And then subclasses with defined attributes for specific purposes that way we could make a "groups" of test cases. This would add `REPRESENTATION_ID` `TEST_PROJECT_NAME` etc for test cases where it's not used at all.

Reply:
Created `testing_classes.py` for this.